homeworks/week2/hw4.js

```diff
@@ -1,5 +1,10 @@
 function printFactor(n) {
-  
+	console.log(1)
```

Comment:
這裡其實可以直接在下面的 for 迴圈從 1 開始就好，不必特別單獨印出 1，大部分的時候儘量讓程式碼是沒有「特例」的會比較好。
貼一個 Linus Torvalds 在一個 [TED 訪談](https://www.youtube.com/watch?v=o8NPllzkFhE&t=858s)裡的說明，裡面的程式碼可以不必看懂（一段 C 語言移除 linked list 某個節點的程式碼），重點是他講的一句話：「…… rewrite it so that a special case goes away and becomes a normal case.」

Reply:
已依建議修改，感謝!